project.xml

```diff
@@ -14,6 +14,7 @@
     <use project = "uuid" optional = "1" implied = "1" />
     <use project = "systemd" optional = "1" implied = "1" min_major = "200" />
     <use project = "lz4" optional = "1" implied = "1" />
+    <use project = "libcurl" optional = "1" impliead = "1" />
```

Comment:
impliead -> implied, this is important to fix as zproject ignores it otherwise

Reply:
oops
one minute